### PR TITLE
feat: add support for deleting configs

### PR DESCRIPTION
**Pull request by ruxandrafed**

### CODE REVIEW

File: sdk/src/sdk.rs [hunk 1]
```diff
@@ -84,6 +84,13 @@ pub fn set_config(key: &str, value: impl Into<serde_json::Value>) -> Result<(),
     )
 }
 
+pub fn delete_config<T: DeserializeOwned>(key: &str) -> Result<T, RequestError> {
+    Ok(serde_json::from_value(sync_request(
+        "config/delete",
+        KeyBody { key },
+  )?)?)
+}
```
Comment (DanielleB-R) [inline sdk/src/sdk.rs:92]: We should be able to make the code simpler by removing the type argument and returning a `Result<bool, RequestError>`. Then the command can just get a boolean value.

Comment (ruxandrafed) [reply sdk/src/sdk.rs:92]: Addressed, thank you!

File: commands/src/config.rs [hunk 2]
```diff
@@ -79,6 +79,33 @@ mod set {
...
+    static KEY: &str = "key";
+
+    pub fn init_cli_command<'a, 'b>() -> App<'a, 'b> {
+        App::new("delete")
+            .about(descriptions::DELETE)
+            .arg(
+                Arg::with_name(KEY)
+                    .short("k")
+                    .long(KEY)
+                    .alias("name")
```
Comment (DanielleB-R) [inline commands/src/config.rs:96]: I realized after looking at some of the other code that we're doing `sdk configs get <KEY>`, so we probably want to make this `sdk configs delete <KEY>`; that will involve replacing these three lines with `.index(1)`

Comment (ruxandrafed) [reply commands/src/config.rs]: Addressed, thank you! https://github.com/cto-ai/sdk-bash/pull/3/commits/6a91629dfdfe4fce5421451970b79a7ea52951a7

**Pull request merged**